src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -101,6 +100,9 @@ EmberEventControl emberAfPluginColorControlServerHueSatTransitionEventControl;
 #define MIN_SATURATION_VALUE 0
 #define MAX_SATURATION_VALUE 254
 #define HALF_MAX_UINT8T 127
+#define HALF_MAX_UINT16T 0x7FFF
```

Comment:
nit: in C++ code we should probably use typesafe constexpr for constants instead of defines